copper-server-api/src/main/java/com/pogeyan/cmis/api/BaseActor.java

```diff
@@ -47,6 +50,7 @@
 	protected static final Logger logger = LoggerFactory.getLogger(BaseActor.class);
 	private Map<String, ActorHandleContext> messageHandles = new HashMap<String, ActorHandleContext>();
 	private Map<String, Timer.Context> perfTimerContext = new HashMap<String, Timer.Context>();
+	private ISpan parentSpan;
```

Comment:
This has to get resolved based on an incoming message, otherwise in parallel calls this will get overriden.